frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.html

```diff
@@ -8,8 +8,11 @@
      [attr.aria-label]="text.query_has_changed_click_to_save"
      [attr.title]="text.query_has_changed_click_to_save">
     <span aria-hidden="true" class="icon icon-save"></span>
-  </a>
+  </a> 
+
   <input #editableTitleInput
+  id="inputTextID"
+  [style.width.pt]="selectedTitle.length > 5 ? (selectedTitle.length + 2)*8 : (selectedTitle.length + 5)*5"
```

Comment:
First of all, please extract the functionality into a method. Second, the switch after 5 characters leads to jump within the text:
![width_toggle](https://user-images.githubusercontent.com/7457313/77404645-566fdf80-6db2-11ea-8894-f72350e1bc66.gif)

And third, these are a lot of magic numbers. Why exactly `+ 2` when the title is long and why `+ 5` when the title is short? Why `* 8` and why `* 5`? Are these numbers necessary? Is the switch really necessary? If yes, where do these numbers come from? Are they the best effort or calculated? This is very hard to understand for anyone who has to read to code later on. So, if they are necessary, please extract them into variables and give them meaningful names to make it easier to understand.